docs/changehistory/NextVersion.md

```diff
@@ -8,17 +8,23 @@ Table of contents:
 - [Display system](#display-system)
   - [Dynamic schedule scripts](#dynamic-schedule-scripts)
   - [Hiliting models and subcategories](#hiliting-models-and-subcategories)
+- [Frontend category APIs](#frontend-category-apis)
```

Comment:
is this supposed to be indented this way?

Reply:
i believe so, this was autocorrected by my md vscode extensions.
this is the parent header for the ones below

see, https://github.com/iTwin/itwinjs-core/blob/webpack5-continued/docs/changehistory/NextVersion.md#frontend-category-apis and look at the display rendered blob 
![image](https://user-images.githubusercontent.com/11051042/181595954-d5ee84b9-2a51-4981-91cd-3046d62f47c8.png)

Reply:
No, it's out of order. I'll fix.

Reply:
Fixed.